archive/oauth-and-openid.md

```diff
@@ -85,3 +156,36 @@ access token.
 
 OIDC is used for **authentication** of the Resource Owner. With ID Token, Client can identify and authenticate
 end-user as well as delegate login to OpenID Provider to enter Partner's endpoint in a secure manner. 
+
+
+# Bearer Token
+
+Bearer token is used for client authentication on OAuth framework. This one way to grant access resource by the
+resource owner. OAuth framework does not require additional credential / authentication with bearer token
+authentication. It means the authentication server need to build their own authentication or access control by
+themselves. Therefore, some of the security considerations are recommended to mitigate threats.
+1. Safeguard bearer token implementation by the server and client
+2. Validate TLS certificate chains and always use TLS (https)
+3. Don't store bearer tokens in cookies
+4. Issue short-lived and scoped bearer tokens
+
+There are three recommended ways to request authentication
```

Comment:
Three implementations are recommended. It is more useful to note the differences. It is hard to understand why there are three methods.

Reply:
`Recommend` gives you a confusion. [rfc6750-authenticated requests](https://datatracker.ietf.org/doc/html/rfc6750) only allows these three ways. I revise the sentence in
```
There are three ways to create authenticated request. Please note that Client MUST NOT user more than one method to transmit the token in each request.
```

Reply:
Aha! I had a misconcept. Thanks for revise. 😃